lib/cmd/common/files.js

```diff
@@ -71,34 +74,33 @@ function responseToFiles(files, file) {
 
 // put our apps into table format..
 function createTableForFiles(filez) {
-  // create our table
   var table = new Table({
     head: ['Name', 'Id'],
     style: common.style()
   });
 
-  // populate our table
-  for (var f in filez) {
-    var id = filez[f];
+  _.each(filez, function (id, f) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)